### ISSUE TITLE
Missing icons from Spyder 5.1.5 (GUI Bug)

### ISSUE THREAD

**Issue by DM20697** [spyder-ide/spyder#18738]
## Problem Description
Hello, I installed the Spyder through Anaconda, but when I opened it, some icons as you can see from the attached figure were missing. I did everything, reset spyder, install from prompt, etc, but nothing. I don't know what to do. Please, if you could help me I would be grateful.
![Καταγραφή_7](https://user-images.githubusercontent.com/109598852/179808034-3e4b79e0-a09b-41f4-b46a-254d3a8f647d.PNG)

## Versions

* Spyder version: 5.1.5
* Python version: 3.9.12
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  8.4.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.17.2 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.4.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<2.10.0        :  2.7.4 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.3.1 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.22.3 (OK)
pandas >=1.1.1                :  1.4.3 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  None (OK)

```

**Comment by ccordoba12** [comment]
Hey @DM20697, thanks for reporting. This error could be fixed in our latest version (5.3.2).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

Please do that and let us know if it works for you.

**Comment by DM20697** [comment]
> Hey @DM20697, thanks for reporting. This error could be fixed in our latest version (5.3.2).
> 
> Since it's still not available in Anaconda, you have two options:
> 
> 1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
> 2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:
>    ```
>     conda create -n spyder-cf -c conda-forge spyder
>     conda activate spyder-cf
>     spyder
>    ```
> 
> Please do that and let us know if it works for you.

Thank you. I did the 2nd method and it worked. I believe that when Anaconda allows the 5.3.2, then I could use the spyder directly.

**Comment by ccordoba12** [comment]
Great! Thanks for letting us know about it.

I'm going to close this issue because there's nothing else we can do on our side, sorry.
